File: app/models/Response.java
package models;

public class Response {
	
	public Response(){}
	
	public Response (String message, int code) {
		this.message = message;
		this.code = code;
	}
	
	public Response (String message, int code, Object result) {
		this.message = message;
		this.code = code;
		this.result = result;
	}

	public String message;
	
	public int code;
	
	public Object result;
	
}

File: app/models/response.rb
class Response

	attr_accessor :message, :code, :result

	def initialize (message = nil, code = 0, result = nil)
		@message = message
		@code = code
		@result = result
	end

	def is_success
		@code == ConstClass::SUCCESS.keys[0]
	end
end